lib/uplink/project.go

```diff
@@ -171,8 +171,7 @@ func (p *Project) OpenBucket(ctx context.Context, bucketName string, access *Enc
 			return nil, err
 		}
 	}
-
-	encryptionScheme := cfg.EncryptionParameters.ToEncryptionScheme()
+	encryptionScheme := cfg.EncryptionParameters
```

Comment:
can we rename this variable encryptionParameters?

Reply:
will rename to EncryptionScheme